.github/CODEOWNERS

```diff
@@ -56,24 +55,25 @@
 /iam/                                  @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra 
 /kms/                                  @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra
 /securitycenter/                       @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra 
+/mediacdn/                             @justin-mp @GoogleCloudPlatform/dee-infra @GoogleCloudPlatform/go-samples-reviewers
 
 # DEE Platform Ops & Observability
 /container/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-platform-ops
-/logging/                              @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-observability 
-/monitoring/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-observability
-/trace/                                @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-observability 
+/logging/                              @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-platform-ops 
+/monitoring/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-platform-ops
+/trace/                                @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-platform-ops 
 
 # Non-DEE owner
+/auth/                                 @GoogleCloudPlatform/googleapis-auth @GoogleCloudPlatform/go-samples-reviewers
```

Comment:
The @GoogleCLoudPlatform/googleapis-auth team isn't a part of this repo.

Reply:
Unfortunately I am not an admin of this repo therefore please add write access to necessary groups per [this comment](https://github.com/GoogleCloudPlatform/golang-samples/pull/2988#issuecomment-1517593799).